src/v/pandaproxy/schema_registry/protobuf.cc

```diff
@@ -131,18 +131,18 @@ struct fmt::formatter<google::protobuf::UninterpretedOption>
         const auto fmt = [&](const auto& val) {
             if (option.has_string_value()) {
                 return fmt::format_to(
-                  ctx.out(), "{} = \"{}\"", fmt::join(option.name(), ""), val);
+                  ctx.out(), "{} = \"{}\"", fmt::join(option.name(), "."), val);
```

Comment:
>pandaproxy/sr: fix protobuf service options rendering

commit message is missing an explanation on why its broken and how this commit fixes it.